### PR TITLE
Started Event API folder and latest yaml

### PR DESCRIPTION
**Pull request by nissimsan**
Hey guys - let me know if this is along the lines of how you think this should be structured. Added a folder for our work and the latest yaml. There's more documentation to come, just wanted to get it started now that we are all focusing on getting git up and running... 
Nis

### CODE REVIEW

**Review by petesavitsky — COMMENTED** [review]
Hey Nis,

This looks cool! Just a couple thoughts on repo structure:

Maybe we can add an `events-api` folder in the top level directory? Then within that, an `events.md` and then a maybe a swagger subfolder where you can put the swagger yml? I'm assuming that's what this is.

 So something like: 
- `event-api/event-api.md`
- `event-api/swagger/event-api.yaml`

The event-api.md could be similar to the other pages on standards and charter, in that it would explain what this is and what it's for.

**Comment by nissimsan** [comment]
For sure, we'll definitely add an .md! 
I'll move the yaml into a swagger subfolder.

**Review by petesavitsky — DISMISSED** [review]

**Comment by nissimsan** [comment]
Added a small change on SMDG too, @petesavitsky. It's something we've discussed in a few meetings. Let me know how you find it...

**Comment by nissimsan** [comment]
(I really need to make some branches - this is getting a bit ugly)  :)

**Comment by nissimsan** [comment]
Alright, done with a baseline of the documentation now...

**Comment by rgenoulaz** [comment]
Looks like it's becoming real.
This is awesome, thank you guys! 
Let's merge this and move forward.

Cheers

Romain

**Review by rgenoulaz — APPROVED** [review]
Lets's go with this first version !
Thanks

File: event-api/event-api.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+This is in contrast to other communication paradigms, notably REST, which is very commonly implemented over APIs, leveraging HTTP’s verbiage for also communicating how data is intended to be stored and manipulated. This is a very practical pattern in many situations, but less suitable for the targeted use cases in the heterogeneous landscape of shipping. Practically, this results in an all HTTP POST API, requiring business logic to properly react. As mentioned above, 1) such business events exist in legacy systems throughout the industry, and 2) significant emphasis has been put on the aspects of genericness, logic and simplicity. 
+
+Please note that the current version of the API is a baseline, not necessarily a final version. It covers the particular scope just described, and there is obviously many other aspects to global transport and trade which may be added in future releases. 
+
+### UN/CEFACT Foundation
+![UNCEFACT Buy-Ship-Pay](images/cefact-bsp.png)
+The Event API allows for publishing Consignments and Transport Equipment as events through a POSTing alongside the events, meant to serves as the context of those subsequent events.
```
Comment (petesavitsky) [inline event-api/event-api.md:26]: `...meant to serve[s] as the context...`

File: event-api/event-api.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+These form a hierarchy, with Transport Equipment sitting underneath a Consignment. Consignment also hold parties. Planned events relate to Consignments; Estimated and Actual events relate to Transport Equipment. 
+From the perspective of common carrier business processes, the Consignment roughly represents the booking confirmation, whereas the Transport Equipment represents the equipment matching (when a physically identified container has been assigned to the consignment). 
+
+Shipment plays an important role, but has not yet been included in this first version of the API entirely due to scope confinement. The Shipment class represents the traded of goods inside the containers, and are of important relevance to authorities, financial institutions, sellers and sellers. It is worth noting that LCL tracking should be done through the Shipment class, so the current version of the API is only meant to deal with FCL. We consider adding the Shipment class a natural future extension of the API.
```
Comment (petesavitsky) [inline event-api/event-api.md:33]: `...represents the [traded of goods?] inside the containers...`

**Review by petesavitsky — APPROVED** [review]
@nissimsan looks good, just a few potential typos in the .md. It would also be cool to stick a link to the swagger API in the .md, I'm assuming you're using an "official" one?

**Pull request merged**